bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -54,6 +56,8 @@
       {{ download_firefox(small=True) }}
       <h1 class="large">{{_('Check Your Plugins')}}</h1>
       <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
+      {% set share_urls = { 'twitter': 'http://mzl.la/17aqnLZ', 'googleplus': 'http://mzl.la/1FuAhTS', 'facebook': 'http://mzl.la/1z4Noqm' } -%}
+      {{ share_cta(_('Share'), share_urls, _('Plugins don’t always update automatically. Check yours now to stay safe and browse without interruption!'), 'spread-plugincheck', 'sky mini') }}
```

Comment:
Oh yes, this page has been localized into many languages. I'll use the `l10n_has_tag` and the preamble for the fallback text:
> Keeping your plugins up to date helps Firefox run safely and smoothly.